dashjoin-rdf4j/src/test/java/org/dashjoin/service/RDF4JPathQLTest.java

```diff
@@ -0,0 +1,126 @@
+package org.dashjoin.service;
+
+import static org.eclipse.rdf4j.model.util.Values.iri;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.ws.rs.core.SecurityContext;
+
+import org.dashjoin.model.AbstractDatabase;
+import org.dashjoin.model.QueryMeta;
+import org.dashjoin.service.Data.Resource;
+import org.dashjoin.util.MapUtil;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTest
+public class RDF4JPathQLTest extends DBTest {
```

Comment:
You don't need to extend DBTest. You can simply annotate the class with QuarkusTest and simply add

  @Inject
  Services services;

  @Inject
  Data db;

Reply:
... but then missing a lot of supporting methods in DBTest such as toID() etc.

Do I need to add additional tests other than testPath()? I only added them because they would fail in DBTest when dj/junit db was used with both junit.nt and junit.additions.nt